gatsby/create-pages/create-carrer-details.js

```diff
@@ -0,0 +1,199 @@
+const fetch = require('node-fetch');
+const { decode } = require('html-entities');
+const xmlParser = require('fast-xml-parser');
+const {
+  PERSONIO_SLUG_FIELD_NAME,
+  extractPersonioSlug,
+} = require('../util/extract-personio-slug');
+const slugify = require('slugify');
+const path = require('path');
+const { siteMetadata } = require('../../gatsby-config');
+const {
+  generateCard,
+} = require('../util/preview-card-generator/generate-card');
+const { appendTrailingSlash } = require('../util/append-trailing-slash');
+
+const CAREER_TEMPLATE_PATH = path.resolve('src/templates/career.tsx');
+const CAREER_DETAILS_TEMPLATE_PATH = path.resolve(
+  'src/templates/career-details.tsx',
+);
+
+const PERSONIO_JOBS_URL = 'https://satellytes.jobs.personio.de/xml';
+const PERSONIO_SHORT_DESCRIPTION_NAME = 'Kurzbeschreibung';
+const LANGUAGES = ['en', 'de'];
```

Comment:
I expected this part to be relevant for other content too, how does it come that we only need that list here?

Reply:
Yep, we also use the languages in the gatsby-config. I will move it there to only have one supported languages field